colors/nord.vim

```diff
@@ -504,6 +504,11 @@ hi! link yamlDocumentStart Keyword
 call s:hi("ALEWarningSign", s:nord13_gui, "", s:nord13_term, "", "", "")
 call s:hi("ALEErrorSign" , s:nord11_gui, "", s:nord11_term, "", "", "")
 
+" Coc
+" > neoclide/coc
+call s:hi("CocWarningSign", s:nord13_gui, "", s:nord13_term, "", "", "")
+call s:hi("CocErrorSign" , s:nord11_gui, "", s:nord11_term, "", "", "")
+
```

Comment:
While we're at it I guess we can also include two of the other custom highlighting groups of the plugin, `CocInfoSign` and `CocHintSign`.
They are [defined in the main `coc-vim` plugin file][defs] and [documented in the plugin's `coc.txt` help file][docs].
I guess using `nord8` for the _info_ sign as foreground color and `nord10` for the _hint_ sign should work fine.

```suggestion
call s:hi("CocInfoSign" , s:nord8_gui, "", s:nord8_term, "", "", "")
call s:hi("CocHintSign" , s:nord10_gui, "", s:nord10_term, "", "", "")

```

[defs]: https://github.com/neoclide/coc.nvim/blob/e88b86eac1dcc1aab9464437d4d2fdeb6b604996/plugin/coc.vim#L181-L182
[docs]: https://github.com/neoclide/coc.nvim/blob/e88b86eac1dcc1aab9464437d4d2fdeb6b604996/doc/coc.txt#L1577-L1588